hexamap.py

```
@@ -128,21 +70,24 @@ def generate_from_files(hexes, output_path, css):
 
     args = parser.parse_args()
 
-    hexfiles = {}
+    metadatas = []
 
     for arg in args.src_path:
         files = glob.glob(arg)
 
         if not files:
             print('File does not exist: ' + arg, file=sys.stderr)
         for file in files:
-            isHexFile, col_file, row_file, contents = parse_hex_file(file)
-            if isHexFile:
-                hexfiles[col_file, row_file] = file, contents
+            # pylint: disable=broad-except
+            try:
+                metadatas.append(TileMetadata.from_file(file))
+            except Exception as e:
+                print(e)
```

Either properly log the exception (import logging) or let it bubble up .

I would definitely vote to let the exception bubble up (if metadatas can't be merged, then result map will be garbled up; hence it would be better to nicely stop further processing

---

No, bubbling up will break the contract, since it's just to avoid unexpected files (not md). I will log it, but maybe it would be good to indroduce some "checker" in tileMetada to avoid having this exception for non-md files.